File: Problem/1285.py
T=int(input())
for test_case in range(1, T+1):
    N=int(input())
    location=list(map(int, input().split()))
    min=100000
    count=0
    for i in range(N):
        if(location[i]<0):
            location[i] = -location[i]

        if(min>location[i]):
            min=location[i]
            count=1
        elif(min==location[i]):
            count+=1
        else:
            pass
    print("#{0} {1} {2}".format(test_case, min, count))

File: Problem/1285.c
#include <stdio.h>
int main(){
    int T, N;
    scanf_S("%d", &T);
    for (int i=1; i<=T; i++){
        scanf_s("%d", &N);
        int min=100000;
        int count=0;
        for(int j=0; j<N; j++){
            int location;
            scanf_s("%d", &location);
            if(location<0){
                location=-location;
            }
            if(min>location){
                min=location;
                count=1;
            }
            else if(min==location){
                count+=1;
            }
            else{}
        }
        printf("#%d %d %d", i, min, count);
    }
}